### ISSUE TITLE
Bug Report: Null object reference while attempting to play Blazin'

### ISSUE THREAD

**Issue by DxrkPxwer** [FunkinCrew/Funkin#2536]
[weed]: <> (FILL THIS ISSUE THING OUT AS MUCH AS POSSIBLE)
[weed]: <> (OR ELSE YOUR ISSUE WILL BE LESS LIKELY TO BE SOLVED!)
[weed]: <> (DO NOT POST ABOUT ISSUES FROM OTHER FNF MOD ENGINES! I CANNOT AND PROBABLY WON'T SOLVE THOSE!)
[weed]: <> (GO TO THEIR RESPECTIVE GITHUB ISSUES AND REPORT THEM THERE LOL!)

[weed]: <> (ALSO MAKE SURE THAT YOU USE PROPER LABELS, IF YOU'RE RUNNING INTO COMPILER ISSUES, USE THE compiler issue LABEL!!!)

### If you are playing the game in a browser, what site are you playing it from?

[weed]: <> (Put an X in the [ ] thingies to fill out checkbox!)
[weed]: <> (something like [x] pretty much, don't screw up or you will look stupid)

- [ ] [Newgrounds](https://www.newgrounds.com/portal/view/770371)
- [ ] [Itch.io](https://ninja-muffin24.itch.io/funkin)? Specify below
- - [x] Windows
- - [ ] Mac
- - [ ] Linux

### If you are playing the game in a browser, what browser are you using?

[weed]: <> (Again, put an x in the [ ] box!)

- [x] None, yes i add this.
- [ ] Google Chrome (or chomium based like Brave, vivaldi, MS Edge)
- [ ] Firefox
- [ ] Safari

## What version of the game are you using? Look in the bottom left corner of the main menu. (ex: 0.2.7, 0.2.1, shit like that)

0.3.3 stuff

## Have you identified any steps to reproduce the bug? If so, please describe them below in as much detail as possible. Use images if possible.

literally just play weekend 1, play 2hot, then watch the cutscene. And BOOM! Null Object Reference.

## Please describe your issue. Provide extensive detail and images if possible.

Yea i just explain that shit
but here is da error log

```
Null Object Reference
  in flixel/graphics/tile/FlxDrawTrianglesItem.hx#63
  in flixel/FlxCamera.hx#774
  in flixel/FlxGame.hx#873
  in flixel/FlxGame.hx#566
  in openfl/events/EventDispatcher.hx#402
  in openfl/display/DisplayObject.hx#1399
```

**Comment by JackXson-Real** [comment]
Are you on Itch.io or your own compiled version? I personally use the Itch.io version and have never had this issue.

**Comment by NotHyper-474** [comment]
Try https://github.com/FunkinCrew/Funkin/issues/2559#issuecomment-2119238934

**Comment by DxrkPxwer** [comment]
i guess ill try to work with that l8er

> Stale issue, I assume this issue is fixed as of v0.5.1.
